main.js

```diff
@@ -1,170 +1,161 @@
-        }   
-      });
+    });
 }
 
-function desbloquearUsuario(element){
+function desbloquearUsuario(element) {
     element.classList.remove("bloqueado");
 }
 
-function turnoUsuario(){
-    escribirMensaje("turno usuario","jugador");
-    $cuadro.forEach(desbloquearUsuario);
-    
-    $cuadro.forEach(function(cuadro){
-        cuadro.onclick = function(e){
-            //cuadro = e.target;
-        e.target.classList.add("activo");
-        setTimeout(function(){
-            e.target.classList.remove("activo");
-        },500);
-        tomarJugadaUsuario(e.target);
-    
-    if($jugadaUsuario.length === $jugadaMaquina.length){
-        
-        compararJugadas($jugadaMaquina, $jugadaUsuario);
-        if(compararJugadas($jugadaMaquina, $jugadaUsuario)==="game over"){
-            alert("perdiste");
-            ronda = 0;
-            escribirMensaje("Clickeá el botón START para comenzar", "perdiste");
-            $jugadaMaquina=[];
-            bloquearUsuario();
-            setTimeout(function(){
-                $start.classList.remove("outmargin");
-            },1000);
-            $bloqueo.classList.remove("bloqueo");
-            ;
-        }
-        else {
-            bloquearUsuario();
-            setTimeout(comenzarJuego, 1000);
+function turnoUsuario() {
+    escribirMensaje("turno usuario", "jugador");
+    $cuadros.forEach(desbloquearUsuario);
+
+    $cuadros.forEach(function (cuadro) {
+        cuadro.onclick = function (e) {
             
+            e.target.classList.add("activo");
+            setTimeout(function () {
+                e.target.classList.remove("activo");
+            }, 500);
+            tomarJugadaUsuario(e.target);
+
+            if (jugadaDeUsuario.length === jugadaDeMaquina.length) {
+
+                if (compararJugadas(jugadaDeMaquina, jugadaDeUsuario) === "game over") {
+                    alert("perdiste");
+                    ronda = 0;
+                    escribirMensaje("Clickeá el botón START para comenzar", "perdiste");
+                    jugadaDeMaquina = [];
+                    bloquearUsuario();
+                    setTimeout(function () {
+                        $start.classList.remove("outmargin");
+                    }, 1000);
+                    ;
+                }
+                else {
+                    bloquearUsuario();
+                    setTimeout(comenzarJuego, 1000);
+
+                }
+            }
         }
-    }
-}
-});
-        
+    });
+
 }
 
 
-function compararJugadas(elemento1, elemento2){
-    const perdiste  = "game over";
+function compararJugadas(elemento1, elemento2) {
+    const perdiste = "game over";
     for (let i = 0; i < elemento1.length; i++) {
-        if(elemento1[i].id===elemento2[i].id){
-            console.log("esta bien!");
-            }
-        else{
+        if (elemento1[i].id === elemento2[i].id) {
+            
+        }
+        else {
             return perdiste;
         }
```

Comment:
No se dejan `ifs` vacíos. En este caso lo que tenés que escribir es la negación
```suggestion
        if (elemento1[i].id !== elemento2[i].id) {
            return perdiste;
        }
```